src/django/api/reports.py

```diff
@@ -120,6 +121,116 @@ def weekly_geocoding_time_with_queue():
+    for product_types in product_type_lists:
+        for p in product_types:
+            data[p.strip()] = data[p.strip()] + 1
+    rows = sort_by_first_column(data.items())
+    return [['product_type_values', 'times_submitted'], rows]
+
+
+def processing_type_facility_type_matched():
+    temp_data = defaultdict(set)
+
+    for (raw_values, matched_values) in ExtendedField.objects.filter(
+        Q(field_name='processing_type') | Q(field_name='facility_type')
+    ).distinct(
+        'value__raw_values', 'value__matched_values'
+    ).values_list('value__raw_values', 'value__matched_values').iterator():
+        values = raw_values
+
+        if isinstance(raw_values, str):
+            values = (raw_values.split('|') if '|' in raw_values
+                      else [raw_values])
+
+        deduped_values = list(dict.fromkeys(values))
+        for i, raw_value in enumerate(deduped_values):
+            if matched_values[i][3] is not None:
+                temp_data[matched_values[i][3]].add(raw_value.strip())
+                temp_data[matched_values[i][2]].add(raw_value.strip())
+
+    data = dict()
+    for (raw_value, match_values) in temp_data.items():
+        data[raw_value] = ' | '.join(list(match_values))
+
+    rows = sort_by_first_column(data.items())
+    return [['taxonomy_value', 'raw_values'], rows]
+
+
+def processing_type_facility_type_unmatched():
+    data = defaultdict(int)
+
+    for (raw_values, matched_values) in ExtendedField.objects.filter(
+        Q(field_name='processing_type') | Q(field_name='facility_type')
+    ).distinct(
+        'value__raw_values', 'value__matched_values'
+    ).values_list('value__raw_values', 'value__matched_values').iterator():
+        values = raw_values
+
+        if isinstance(raw_values, str):
+            values = (raw_values.split('|') if '|' in raw_values
+                      else [raw_values])
+
+        deduped_values = list(dict.fromkeys(values))
+        for i, raw_value in enumerate(deduped_values):
+            if matched_values[i][3] is None:
+                data[raw_value.strip()] = data[raw_value.strip()] + 1
+
+    rows = sort_by_first_column(data.items())
+    return [['processing_type_facility_type', 'times_submitted'], rows]
+
+
+def contributors_with_extended_fields_cumulative():
+    data = defaultdict(list)
+    months = ExtendedField.objects.dates('created_at', 'month')
+    for month in months:
+        m = month.strftime(DateFormats.MONTH)
+        data[m] = [m]
+        fields = ExtendedField.objects.filter(
+            created_at__year__lte=month.year,
+            created_at__month__lte=month.month)
+
+        for field_name, _ in ExtendedField.FIELD_CHOICES:
+            field_count = fields.filter(field_name=field_name).distinct(
+                'contributor_id').count()
+            data[m].append(field_count)
+
+    columns = ['month']
+    for field_name, _ in ExtendedField.FIELD_CHOICES:
+        columns.append(field_name)
```

Comment:
The OAR team request that we add a "(claim)" suffix to the `country`, `name` and `address` column headers to clearly indicate that they come from the claim process, bot facility data submission.

Reply:
Followup question: We had added `country` as one of the initial fields, but I don't believe that the claim form actually includes `country` as a value for the facility (only for the Office Information section). Do we want to remove this as well as the two duplicate values, since it's not actually a possible field type at the moment?

Reply:
Good idea. Having the country column filled with all zeros all the time just adds noise to the report.